Sources/NetworkProtection/Routing/VPNRoutingTableResolver.swift

```diff
@@ -53,7 +53,7 @@ struct VPNRoutingTableResolver {
         var routes = VPNRoutingRange.publicNetworkRange + dnsRoutes()
 
         if !excludeLocalNetworks {
-            routes += VPNRoutingRange.localNetworkRange
+            routes += VPNRoutingRange.localNetworkRangeWithDNS
```

Comment:
Feel free to ignore cos this is nitpicky.   actually think the names should be the the opposite, because technically speaking 10/8 is part of the local network range.  ie:

`localNetworkRange` > `localNetworkRangeExceptDNS`
`localNetworkRangeWithDNS` > `localNetworkRange`

Reply:
This is a great point, completely agreed.